pkg/cluster/formatter.go

```diff
@@ -55,7 +55,41 @@ type MachineStatus struct {
 func (JSONFormatter) Format(w io.Writer, machines []*Machine) error {
 	var statuses []MachineStatus
 	for _, m := range machines {
-		statuses = append(statuses, *m.Status())
+		s := status{}
```

Comment:
I  moved the `Status` computation in on the Machine object in `pkg/cluster`, that's why it was conflicting. The right fix is to fix the code in `Machine.Status()`.